$OpenBSD: patch-src_client_components_conversation-find-bar_vala,v 1.3 2017/04/13 07:10:21 jasper Exp $

Fix build with vala 0.36:
error: Argument 1: Cannot convert from `uint' to `int'

--- src/client/components/conversation-find-bar.vala.orig	Thu Apr 13 08:57:10 2017
+++ src/client/components/conversation-find-bar.vala	Thu Apr 13 09:00:07 2017
@@ -211,9 +211,9 @@ public class ConversationFindBar : Gtk.Layout {
         
         if (matches > 0) {
             if (!wrapped)
-                content += ngettext("%i match", "%i matches", matches).printf(matches);
+                content += ngettext("%i match", "%i matches", matches).printf((int) matches);
             else
-                content += ngettext("%i match (wrapped)", "%i matches (wrapped)", matches).printf(matches);
+                content += ngettext("%i match (wrapped)", "%i matches (wrapped)", matches).printf((int) matches);
         } else {
             content += _("not found");
         }
